clang/include/clang/Basic/DiagnosticFrontendKinds.td

```diff
@@ -226,6 +226,7 @@ def err_module_map_not_found : Error<"module map file '%0' not found">,
 def err_missing_module_name : Error<
   "no module name provided; specify one with -fmodule-name=">,
   DefaultFatal;
+def err_file_is_not_module : Error<"file '%0' is not a module">, DefaultFatal;
```

Comment:
```suggestion
def err_file_is_not_module : Error<"file '%0' is not a module file">, DefaultFatal;
```

Reply:
Done. Extended the description

Reply:
@ChuanqiXu9 FYI i don't have commit access so when PR is ready please commit it

Reply:
OK, I'll commit it.